content/en/docs/reference/glossary/namespace-sameness.md

```diff
@@ -5,5 +5,5 @@ test: n/a
 
 Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
 applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
-`Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
-load-balanced across all clusters in the mesh for a given service.
\ No newline at end of file
+`Service` with the same namespaced name and the same port name, then they will be recognized as a single combined service. By default, traffic is
```

Comment:
This seems a bit misleading. It seems to be saying that "service with same port name", which suggests a service has 1 port - it can have many. Additionally, ports not matching doesn't mean the whole _Service_ is not considered 'same'; rather, just the port is not considered.

Reply:
Thanks for reply.
Sorry, my words did misleading. Let me give a example:

```yaml
# the one in primary cluster
apiVersion: v1
kind: Service
metadata:
  name: nginx-service
spec:
  selector:
    app.kubernetes.io/name: proxy
  ports:
  - name: primary-cluster-80
    protocol: TCP
    port: 80
    targetPort: http-web-svc

# the one in a remote cluster
apiVersion: v1
kind: Service
metadata:
  name: nginx-service
spec:
  selector:
    app.kubernetes.io/name: proxy
  ports:
  - name: remote-cluster-80
    protocol: TCP
    port: 80
    targetPort: http-web-svc
  - name: remote-cluster-irrelevant-port
    protocol: TCP
    port: 8080
    targetPort: irrelevant-port
```

These two service, in same namespace, had same name, both had 80 port, the one in remote cluster, also had an extra port, that's OK, service can have many ports.

But, for the 80, these two service had different `port.name`, one is `primary-cluster-80`, the other is `remote-cluster-80`.
If so, the eds of cds `outbound|80||nginx-service.default.svc.cluster.local` only contains endpoints in primary cluster, include the eastwest gateway. Pods in remote make request to this service, will always goes to workloads in primary cluster.

I will try to choose other words and ask for some help around, to express this more clear.

Reply:
Yeah I argee, the poit I was trying to make is the granularity is Port, not Service -- if one of the port names matched they would still be counted